modules/login/templates/form_passwordexpiry.tpl

```diff
@@ -7,8 +7,8 @@
       <p><b>Password Strength Rules</b></p>
       <ul>
         <li>The password must be at least 8 characters long.</li>
-        <li>The password and the user name must not be the same.</li>
         <li>The password and the email address must not be the same.</li>
+        <li>No special characters are required but your password must be sufficiently complex to be accepted.</li>
```

Comment:
Can you clarify what are the condition for "sufficiently complex" password?

Someone might think that `9876SA321` is enough

Reply:
Its not really possible to provide strict requirements as those are mostly meaningless compared to the entropy calculations provided by Zxcvbn.

When Zxcvbn is fully integrated on the frontend it will provide live feedback on on the password and show when it "passes" the minimum requirements.

Reply:
@PapillonMcGill see my above comment. There isn't really a way to explain it better. I agree we really need the front-end Zxcvbn module #5761 which is held up by #5828

Reply:
@johnsaigle maybe add 
"... sufficiently complex and hard to guest to be accepted."

Reply:
What is "hard to guess" though?

Reply:
I know, not easy. Ok, let go with it, better than current.